File: src/main/java/cn/xihan/lib/easyhttp/request/PatchRequest.java
package cn.xihan.lib.easyhttp.request;


import cn.xihan.lib.easyhttp.model.HttpMethod;

/**
 *    author : Android 轮子哥
 *    github : <a href="https://github.com/getActivity/EasyHttp">...</a>
 *    time   : 2020/10/07
 *    desc   : Patch 请求
 */
public final class PatchRequest extends BodyRequest<PatchRequest> {

    public PatchRequest() {
        super();
    }

    @Override
    public String getRequestMethod() {
        return HttpMethod.PATCH.toString();
    }
}

File: src/main/java/cn/xihan/lib/easyhttp/request/PatchRequest.kt
package cn.xihan.lib.easyhttp.request

import cn.xihan.lib.easyhttp.model.HttpMethod

/**
 * author : Android 轮子哥
 * github : [https://github.com/getActivity/EasyHttp](https://github.com/getActivity/EasyHttp)
 * time   : 2020/10/07
 * desc   : Patch 请求
 */
class PatchRequest : BodyRequest<PatchRequest?>() {
    override val requestMethod: String
        get() = HttpMethod.PATCH.toString()
}